rfcs/README.md

```diff
@@ -0,0 +1,139 @@
+
+In order to accept your pull request, we need you to submit a CLA. You only need
+to do this once, so if you've done this for another Elastic open source
+project, you're good to go.
+
+**[Complete your CLA here.](https://www.elastic.co/contributor-agreement)**
+
+## When to follow this process
+
+You should consider using this process if you intend to make "substantial"
+changes to Kibana or its documentation. Some examples that would benefit
+from an RFC are:
+
+   - A new feature that creates new API surface area, such as a new core
+     service available to plugins.
+   - The removal of features that already shipped as part of a release.
+   - The introduction of new idiomatic usage or conventions, even if they
+     do not include code changes to Kibana itself.
+
+The RFC process is a great opportunity to get more eyeballs on your proposal
+before it becomes a part of a released version of Kibana. Quite often, even
+proposals that seem "obvious" can be significantly improved once a wider
+group of interested people have a chance to weigh in.
+
+The RFC process can also be helpful to encourage discussions about a proposed
+feature as it is being designed, and incorporate important constraints into
+the design while it's easier to change, before the design has been fully
+implemented.
+
+Some changes do not require an RFC:
+
+  - Rephrasing, reorganizing or refactoring
+  - Addition or removal of warnings
+  - Additions that strictly improve objective, numerical quality
+  criteria (speedup, better browser support)
+  - Addition of features that do not impact other Kibana plugins (do not
+  expose any API to other plugins)
+
+## What the process is
+
+In short, to get a major feature added to Kibana Core, one usually first gets
+the RFC merged into the RFC tree as a markdown file. At that point the RFC
+is 'active' and may be implemented with the goal of eventual inclusion
+into Kibana.
+
+* Fork the Kibana repo http://github.com/elastic/kibana
+* Copy `rfcs/0000_template.md` to `rfcs/text/0000_my_feature.md` (where
+'my_feature' is descriptive. Don't assign an RFC number yet).
+* Fill in the RFC. Put care into the details: **RFCs that do not
+present convincing motivation, demonstrate understanding of the
+impact of the design, or are disingenuous about the drawbacks or
+alternatives tend to be poorly-received**.
+* Submit a pull request. As a pull request the RFC will receive design
+feedback from the larger community and Elastic staff. The author should
+be prepared to revise it in response.
+* Build consensus and integrate feedback. RFCs that have broad support
+are much more likely to make progress than those that don't receive any
+comments.
+* Eventually, the team will decide whether the RFC is a candidate
+for inclusion in Kibana.
+* RFCs that are candidates for inclusion in Kibana will enter a "final comment
+period" lasting 3 calendar days. The beginning of this period will be signaled with a
```

Comment:
```suggestion
period" lasting at least 3 working days. The beginning of this period will be signaled with a
```

Most of the development happens while people are working, so we want to give folks enough time.